### DISCUSSION TITLE
useBridge/Bridge, how and when do you use it?

### DISCUSSION THREAD

**Discussion by github0013** [pmndrs/jotai#292] [Q&A] [answered]
https://github.com/pmndrs/jotai/tree/master/docs/api
I read through API documents, and I pretty much understood them except useBridge/Bridge.  
I even checked the codesandbox example, but I still can't figure out what it is doing for any good reasons.

Can someone explain what it is good for?

**Comment by dai-shi** [comment, marked as answer]
You don't need them unless you need them. 😄 
It's introduced for react-three-fiber and for other custom renderers / multiple roots.
Here's a reference: https://github.com/pmndrs/react-three-fiber/blob/master/markdown/api.md#gotchas

We should add some notes in https://github.com/pmndrs/jotai/blob/master/docs/api/core.md#usebridgebridge 🤔

**Reply by github0013** [reply]
The problem is I can't even tell if I need them because I don't understand what they do...
I really appreciate if you could add some notes. 

Thanks
